src/hooks/chart.ts

```diff
@@ -131,11 +132,17 @@ export type PlotTemplateParams = {
     [paramKey: string]: any;
   };
   /**
-   * Parameters for URL
+   * Parameters for URL //Deprecated. Use control_values instead
```

Comment:
This actually won't be deprecated anymore. We are going to support `url_parameters` in the case that one is defined that is NOT used as part of a control on the page.

For instance, a query parameter is passed to the app that is directly used in the query and can't be changed by the app.

All query parameters will have their `value` stored in `$url_parameters.<key> = <value>` on app initialization. For example, we have `/deriva-webapps/plot/?config=barplot&consortium=ALL` which would have the following `$url_parameters`:
```
$url_parameters: {
  config: 'barplot',
  consortium: 'ALL'
}
```

Reply:
Got it! I have made changes.

Reply:
Regarding the comment `whitespace between the navbar and controls should be reduced` : This was due to the container_padding I've commented out the parameter from sample_config so that you can test with that